UI/index.html

```diff
@@ -0,0 +1,245 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
There is no tag `<meta name="viewport" content="width=device-width, initial-scale=1.0">`, so it looks bad on mobile